src/MoBi.Core/Domain/Model/MoBiSimulation.cs

```diff
@@ -74,7 +74,7 @@ public bool IsCreatedBy(IBuildingBlock templateBuildingBlock)
 
       public bool UsesObservedData(DataRepository dataRepository)
       {
-         return Charts.Any(x => chartUsesObservedData(dataRepository, x));
+         return OutputMappings.Any(x => x.UsesObservedData(dataRepository)) || Charts.Any(x => chartUsesObservedData(dataRepository, x));
```

Comment:
I think this is a good way to check if an observed data is used. Either it's used in the chart or it's referenced in the output mappings.